### DISCUSSION TITLE
Running workflow using remote desktop results in crash

### DISCUSSION THREAD

**Discussion by AOONIH** [bonsai-rx/bonsai#925] [Q&A]
Starting a workflow on a machine via remote desktop (machine running bonsai is Win7) causes windows error reporting bonsai crash with no runtime error or error message from bonsai.

**Comment by glopesdev** [comment]
I've seen similar feedback before, but wasn't able to reproduce/debug the situation yet. Have you tried launching Bonsai from the command-line to see if any error traces pop-up when it crashes?

**Comment by jlarsch** [comment]
I also have this issue. It happens for workflows that open or close a visualizer.
When you log onto a computer that is already running a workflow with an open visualizer, that's OK. But once you stop the running workflow via remote desktop, bonsai crashes. The same happens when starting a workflow that opens a visualizer.
One suboptimal workaround for me has been to deactivate all visualizers before starting a workflow remotely.
